PyQt5-stubs/QtCore.pyi

```diff
@@ -12297,6 +12297,67 @@ class QSysInfo(sip.simplewrapper):
 
     WordSize = ...  # type: QSysInfo.Sizes
 
+    class WinVersion(int):
```

Comment:
I guess this should be present if we built the upstream on Windows?  Which raises an annoying question about how to handle upstream updates on multiple platforms.  `:[`

Reply:
I guess so. But anyway, teststub is very good at catching this. So, you already have an answer : we need to make sure to have CI run on all platforms.

In this case, these are deprecated functions, so no worry.